src/modules/layout/containers/PermissionSnackbarContainer.js

```diff
@@ -30,6 +31,7 @@ class PermissionSnackbarContainer extends React.Component<PropsType, StateType>
   constructor (props: PropsType) {
```

Comment:
This would become a `SnackbarContainer` und not a `PermissionSnackbarContainer` anymore.